src/main/java/com/cinema/model/dto/CinemaHallResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class CinemaHallResponseDto {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```
the default value of long is 0L